core/key.go

```diff
@@ -384,7 +404,7 @@ func (k CollectionSchemaKey) ToDS() ds.Key {
 }
 
 func (k CollectionSchemaVersionKey) ToString() string {
-	result := COLLECTION_SCHEMA_VERSION
+	result := COLLECTION_SCHEMA_VERSION + "/" + string(ValueKey)
```

Comment:
Note: I am reusing the `InstanceType` values (directly) here, I like the consistency and think it fits, but I can also see how it may be seen as pointless coupling and abuse - do let me know :)

Reply:
Can't you explain why you want to use the value key in the path? At first glance it seems unnecessary.

Reply:
Not a problem, will add a comment (and on the history key which does the same)

Reply:
Unless there's a necessity for this I don't see, I would suggestion removing it.

Reply:
I thought about this a bit more, I'm going to split the const in to `/collection/version/v` and `/collection/version/h` (dropping the runtime concat of the InstanceType).  It has tripped two readers up slightly, and you cant see the reason for it reading each ToString implementation in isolation (without comment).  As the consts will be declared next to each other they should be pretty obvious as to why they exist too.

- [ ] Split const, rm use of InstanceType